addon/templates/components/power-select/options.hbs

```diff
@@ -5,18 +5,29 @@
 {{/if}}
 {{#each options as |opt index|}}
   {{#if (ember-power-select-is-group opt)}}
-    <li class="ember-power-select-group" aria-disabled={{ember-power-select-true-string-if-present opt.disabled}} role="option">
-      <span class="ember-power-select-group-name">{{opt.groupName}}</span>
-      {{#component optionsComponent
-        options=(readonly opt.options)
-        select=(readonly select)
-        groupIndex=(concat groupIndex index ".")
-        optionsComponent=(readonly optionsComponent)
-        role="group"
-        class="ember-power-select-options" as |option|}}
-        {{yield option select}}
-      {{/component}}
-    </li>
+    {{#with
```

Comment:
I'm still jet-lagged AF, but wouldn't be easier to just render the default group component?

```hbs
{{#if (ember-power-select-is-group opt)}}
  {{#component groupComponent 
    group=opt 
    select=select
    optionsComponent=(readonly optionsComponent) as |option|}}
    {{yield option select}}
  {{/component}}
{{else}}
```

I see that with this approach the custom component only modifies the group, no the options. I like that tho. I'll review it again this week once I have a functional brain.